wcfsetup/install/files/ts/WoltLabSuite/Core/Dom/Util.ts

```diff
@@ -472,6 +472,23 @@ const DomUtil = {
 
     return innerError as HTMLElement | null;
   },
+
+  /**
+   * Finds the closest element that matches the provided selector. This is a helper
+   * function because `closest()` does exist on elements only.
```

Comment:
```suggestion
   * function because `closest()` does exist on elements only (i.e. it is missing on text nodes).
```
?

Note: Applying this suggestion probably makes Prettier fail.